src/yunohost/app.py

```diff
@@ -828,6 +831,9 @@ def confirm_install(confirm):
         if os.path.exists(os.path.join(extracted_app_folder, file_to_copy)):
             os.system('cp -R %s/%s %s' % (extracted_app_folder, file_to_copy, app_setting_path))
 
+    # Create permission before the install (useful if the install script redefine the permission)
+    permission_add(auth, app=app_instance_name, permission="main")
```

Comment:
Any chance that we can add `sync_perm=False` here ? Like, to avoid triggering a whole bunch of code and messages ? I dunno what are the implications tho :s

Reply:
Should be possible I think. The only issue that I think we can if with this idea is that the application try to update the permission and not everything is synced. But I not sure about if it's possible and if yes it should be probably a really edge case...

So I would say yes we can add `sync_perm=False` and if we have some issue we can remove it.

Reply:
Done with some comment to explain the question (d0f422ee)